examples/runCfhtQuickTest.sh

```diff
@@ -7,12 +7,45 @@ CAMERA=CfhtQuick
 CONFIG_FILE="${PRODUCT_DIR}/config/cfhtConfig.py"
 MAPPER=lsst.obs.cfht.MegacamMapper
 
-"${PRODUCT_DIR}/examples/processData.sh" \
-    -c "$CAMERA" \
-    -m "$MAPPER" \
-    -v "$VALIDATION_DATA_DIR" \
-    -f "$CONFIG_FILE" \
-    -- "$@"
-"${PRODUCT_DIR}/examples/validateRepo.sh" \
-    -c "$CAMERA" \
-    -- "$@"
+print_error() {
+    >&2 echo "$@"
+}
+
+DOPROCESS=true
+DOVERIFY=true
+
+usage() {
+    print_error
+    print_error "Usage: $0 [-pv] [-h] [-- <extra options to validateDrp.py>]"
+    print_error
+    print_error "Specifc options:"
+    print_error "   -p          Skip processing?"
+    print_error "   -v          Skip verification?"
+    print_error "   -h          show this message"
+    exit 1
+}
+
+# thank OSX for not including getopt
+while getopts "vp" option; do
+    case "$option" in
+        p)  DOPROCESS=false;;
+        v)  DOVERIFY=false;;
+        h)  usage;;
+        *)  usage;;
+    esac
+done
+shift $((OPTIND-1))
+
+if [ "$DOPROCESS" = true ] ; then
```

Comment:
Since the shebang line is invoking bash, the improved `[[ ]]` can be used for testing.  Variables do not need to be quoted inside of `[[ ]]` and `==` may be used instead of the posix `=` equality operator.

Reply:
Will do.